busylight/api/busylight_api.py

```diff
@@ -6,11 +6,14 @@
 from typing import Callable, List, Dict, Any
 
 from fastapi import Depends, FastAPI, HTTPException, Path, Request, status
+from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse
 from fastapi.security import HTTPBasic, HTTPBasicCredentials
+from json import loads as json_loads
 from loguru import logger
 
 from .. import __version__
+from ..__main__ import GlobalOptions
```

Comment:
I admit I don't understand what's going on here. There isn't a `GlobalOptions` defined in `__main__` that I can see.

Reply:
I found it at https://github.com/mchaker/busylight/blob/add-cors-middleware/busylight/__main__.py#L27

(`busylight/__main__.py`, L27)

Reply:
SMH - I'm switching between the new light implementation and this pull request. Nevermind, carry on.